src/lib/plugin.js

```diff
@@ -808,9 +808,8 @@ class FiveBellsLedger extends EventEmitter2 {
   }
 
   _requestRetry (requestOptions, retryOptions) {
-    return requestRetry(Object.assign({
-      credentials: this.credentials
-    }, requestOptions), retryOptions)
+    return requestRetry(Object.assign(requestCredentials(this.credentials), requestOptions),
```

Comment:
Object.assign's should have an extra first argument of `{}`, because it will mutate the first argument